src/rp2040/hardware_structs/include/hardware/structs/interp.h

```diff
@@ -8,17 +10,34 @@
 #define _HARDWARE_STRUCTS_INTERP_H
 
 #include "hardware/address_mapped.h"
-#include "hardware/platform_defs.h"
 #include "hardware/regs/sio.h"
 
+// reference to datasheet: https://datasheets.raspberrypi.org/rp2040/rp2040-datasheet.pdf#tab-registerlist_sio
+// BITMASK : FIELDNAME [BITRANGE] (RESETVALUE): DESCRIPTION
+
 typedef struct {
+    _REG_(SIO_None_OFFSET)
     io_rw_32 accum[2];
+
+    _REG_(SIO_None_OFFSET)
     io_rw_32 base[3];
```

Comment:
Is the `_None_` here an indication of a bug in the Python code that generates these files?

Reply:
Yes..xD

Reply:
Also now fixed